tests/queries/tests.py

```diff
@@ -3803,3 +3804,36 @@ def test_ticket_23622(self):
             Ticket23605A.objects.filter(qx),
             [a2], lambda x: x
         )
+
+
+class TestMetaTableClass(TestCase):
+    @unittest.skipUnless(connection.vendor == 'postgresql', 'PostgreSQL specific DDL used')
+    def test_schema_qualified(self):
+        cursor = connection.cursor()
+        cursor.execute('create schema other_schema')
+        cursor.execute('create table other_schema.schema_qualified(id serial primary key, val text)')
```

Comment:
No, and I don't intend to do that for the initial merge. Just creating tables doesn't require much changes to migrations. I guess the biggest problems are:
  - Creation of the schema itself
  - We need some support for db inspection (IIRC migrations check which tables exists)
  - What to do with SQLite which doesn't support schemas at all

Same table in multiple schemas requires a lot more changes, but I don't think we have to solve that one at all.